src/C1T2X_OBU.py

```diff
@@ -178,7 +199,7 @@ def main():
 	threads.append(VANET_mt)
 
 	for thread in threads:
-		c1t2x_logger.debug("Starting %s" %thread.name)
+		c1t2x_logger.debug("Starting {}".format(thread.name))
```

Comment:
This should be refactored to use the optimized logging syntax. Link [here](https://docs.python.org/3/howto/logging.html#optimization). Please change for all applicable.